packages/conf-llvm/conf-llvm.3.9/files/source-linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main'
+curl -L http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Mmmh, looking at the new CI (ci.ocaml.io), it seems that it runs Alpine Linux so I just pushed a commit that removes the source scripts and give the alpine packages. We'll see if it's enough :)